test/known_issues/test-child-process-exec-stdout-data-string.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/7342
+const common = require('../common');
+const assert = require('assert');
+const exec = require('child_process').exec;
+
+const keepAlive = setInterval(() => {}, 9999);
```

Comment:
Won't the child process keep the event loop open? Works for me locally without the timer.